packages/common/container-definitions/package.json

```diff
@@ -100,7 +100,7 @@
 		"@fluid-tools/build-cli": "^0.40.0",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.40.0",
-		"@fluidframework/container-definitions-previous": "npm:@fluidframework/container-definitions@2.0.0-rc.5.0.0",
+		"@fluidframework/container-definitions-previous": "npm:@fluidframework/container-definitions@2.0.0",
```

Comment:
Noting for follow up - find the commit which would have added the "broken" entries.